assets/css/main.css

```diff
@@ -0,0 +1,333 @@
+}
+
+.first-nav {
+  background-color: #3e3e3e;
+}
+
+.first-footer {
+  color: #9c9c9c;
+}
+
+.mobMenu {
+  height: 100vh;
+  display: none;
+  flex-direction: column;
+}
+
+.mobMenu > ul > li {
+  display: flex;
+  margin-top: 40px;
+  margin-right: 32px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#closeIcon {
+  text-decoration: none !important;
+}
+
+.closeIcon {
+  margin-top: 16px;
+  margin-right: 28px;
+  color: black;
+  cursor: pointer;
+}
+
+.text_imgback {
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-image: url('../img/main.jpg');
+  background-size: 300% 300% !important;
+  -webkit-animation: gradient 10s ease-in-out infinite;
+  -moz-animation: gradient 10s ease-in-out infinite;
+  animation: Gradient 10s ease-in-out infinite;
+  margin: 0;
+  font-size: 40px;
+}
+
+.text_header {
+  font-size: 30px;
+  padding: 0;
+  margin: 0;
+}
+
+.section1 {
+  background-image: url("../img/bg.jpg");
+  background-color: #d3d3d3;
+  background-blend-mode: lighten;
+}
+
+.text-lato {
+  font-family: 'Lato', sans-serif;
+}
+
+.text-cocogoose {
+  font-family: 'Cocogoose', sans-serif;
+}
+
+.text-darkp {
+  color: #272a31 !important;
+  font-family: 'Lato', sans-serif;
+}
+
+.text-magenta {
+  color: #ec5242;
+  font-family: 'Lato', sans-serif;
+}
+
+.bg-magenta {
+  background-color: #ec5242 !important;
+}
+
+.bg-light {
+  background-color: #d3d3d3;
+}
+
+.bg-light-transparent {
+  background-color: rgba(255, 255, 255, 0.2);
+}
+
+.section2 {
+  background: linear-gradient(-45deg, #222 50%, transparent 5%);
+  background-size: 20px 20px;
+  background-color: #272a31;
+}
+
+.w-100 {
+  width: 100%;
+}
+
+.paragraph {
+  height: 236px;
+}
+
+.paragraph1 {
+  font-size: 12px;
+  margin-top: 1rem !important;
+  margin-bottom: 1rem !important;
+}
+
+.psubtitle1 {
+  font-size: 20px;
+  margin-bottom: 0 !important;
+}
+
+.psubtitle2 {
+  font-size: 15px;
+}
+
+.separation1 {
+  padding-top: 100px;
+}
+
+.h5style1 {
+  font-size: 25px;
+}
+
+.h5style2 {
+  font-size: 16px;
+  padding-top: 16px;
+}
+
+.h5style3 {
+  font-size: 14px;
+  font-weight: bold;
+  padding-top: 35px;
+  padding-bottom: 15px;
+}
+
+.h5style4 {
+  font-size: 13px;
+}
+
+.divstyle {
+  font-size: 20px;
+  text-decoration: underline;
+}
+
+.mainstyle {
+  padding: 0.8rem !important;
+}
+
+.h1style {
+  font-size: 56px;
+}
+
+.paragraph2 {
+  font-size: 18px;
+}
+
+.imgstyle {
+  width: 100%;
+  height: auto;
+}
+
+.members1 {
+  font-size: 13px;
+}
+
+.members2 {
+  font-size: 11px;
+}
+
+.members3 {
+  font-size: 8px;
+}
+
+@media (min-width: 410px) and (max-width: 768px) {
+  .members3 {
+    font-size: 14px;
+  }
+}
+
+@media (max-width: 768px) {
+  .hidden-mobile {
+    display: none;
+  }
+
+  header {
+    position: static;
+    padding-left: 24px;
+    padding-right: 24px;
+    box-shadow: none;
+  }
+}
+
+@media (min-width: 768px) and (max-width: 1100px), (min-width: 1100px) and (max-width: 2024px) {
```

Comment:
[OPTIONAL] I think this media query might be a bit redundant. Remember that when we're using the "Mobile First approach" design we want to use just as much as possible just `min-width` queries. Therefor, I would rewrite this as `@media (min-width: 768px)`. That's all you need, all of this styles will apply from 768 to "infinite", if ever you want to target bigger screens, you could add next breakpoint just the same. e.g.: `@media (min-width: 1024px)`.

[Here is another example and a link to the original article.](https://zellwk.com/blog/how-to-write-mobile-first-css/)
![image](https://user-images.githubusercontent.com/79114233/142354785-a71eed0c-e8ba-43eb-8ef9-6ec9c271212b.png)

Reply:
Thank you @D4nielJ for your feedback. Your observation is now implemented on commit 0196cdb.